### ISSUE TITLE
💬 Allow mention users in the comments

### ISSUE THREAD

**Issue by jakobroehrl** [stefan-niedermann/nextcloud-deck#673]
In the browser the @ sign shows up a list of users to mention. 
This function would be nice to have here, too.

**Comment by stefan-niedermann** [comment]
I think we need support in the server app before we can implement it. The `description` property includes more meta data about the mentions, we need this stuff for the comments, too.

**Comment by jakobroehrl** [comment]
cc @juliushaertl

**Comment by stefan-niedermann** [comment]
Forget it, i didn't read the issue correctly. I am dumb. We don't need anything from the server, sorry for the noise. 😳

**Comment by stefan-niedermann** [comment]
@desperateCoder i need some regex magic for `if <anything-or-empty>@<anything-except-whitespaces-or-enter-or-space> then true`

**Comment by juliushaertl** [comment]
As a small hint, mentions also might be like `@"User Name" if the uid in Nextcloud contains spaces.

**Comment by desperateCoder** [comment]
Great hint @juliushaertl, thanks! Additionally we wondered, what exactly is valid for a uid. Are `-`, `_` or any other special characters allowed?

**Comment by desperateCoder** [comment]
@stefan-niedermann have a look. Also see `CommentsUtilTest` and add, whatever you can imagine.

**Comment by juliushaertl** [comment]
https://github.com/nextcloud/server/blob/cb057829f72c70e819f456edfadbb29d72dba832/lib/private/Comments/Comment.php#L231 might help ;)

**Comment by stefan-niedermann** [comment]
Will be shipped in `1.10.0`
